### PR TITLE
feat: email magic link in development

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
- feat: send the magic link via email
- feat: wrap the link in an anchor tag

With https://github.com/freeCodeCamp/chapter/issues/445 in mind, this makes the login process a little more natural.

### CODE REVIEW

**Comment by ojeytonwilliams** [comment]
Just to clarify, this should make the dev experience a bit smoother and provide a fallback for the demo.  The plan is still to provide Google auth for the demo and a few more options for the MVP.

File: server/controllers/Auth/resolver.ts [hunk 1]
```diff
@@ -52,6 +53,11 @@ export class AuthResolver {
       console.log(
         `Code: ${code}\nhttp://localhost:3000/auth/token?token=${token}`,
       );
+      new EmailResolver().sendEmail({
+        to: [data.email],
+        subject: 'Login to Chapter',
+        html: `<a href=http://localhost:3000/auth/token?token=${token}>Click here to log in to chapter</a>`,
+      });
```
Comment (Zeko369) [inline server/controllers/Auth/resolver.ts:60]:
```suggestion
      await new MailerService(
        [data.email],
        'Login to Chapter',
        `<a href=http://localhost:3000/auth/token?token=${token}>Click here to log in to chapter</a>`,
      ).sendEmail();
```

**Review by Zeko369 — CHANGES_REQUESTED** [review]
I wouldn't mix resolvers into resolvers, create a new service here

**Comment by ojeytonwilliams** [comment]
> I wouldn't mix resolvers into resolvers, create a new service here

Thanks, I wasn't entirely confident this was right. Is the point that resolvers should just be concerned handling GraphQL queries/mutations?

**Comment by Zeko369** [comment]
Yup, when you annotate a class with `@Resolver` use it only in the typeGraphql context, because other stuff might break (args, ctx, other decorators)

**Review by Zeko369 — APPROVED** [review]

**Pull request merged**